api/src/controllers/permission.ts

```diff
@@ -71,7 +73,7 @@ export interface PermissionDetailsResponse {
 @Tags('Permission')
 export class PermissionController {
   /**
-   * @summary Get list of all permissions (uri, setting and userDetail).
+   * @summary Get a list of user's permissions, if user is admin all permissions are returned.
```

Comment:
Do we have a 'details' section?

Summary: Get the list of permitted routes

Details:  Get the list of permitted routes available to the authenticated user.  If the user is an admin, all routes are returned.  In future, this endpoint will also include the permitted files & folders.